handler.go

```diff
@@ -189,57 +189,23 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 
 // serveWriteSeries receives incoming series data and writes it to the database.
 func (h *Handler) serveWriteSeries(w http.ResponseWriter, r *http.Request, u *User) {
-	// TODO: Authentication.
+	database := r.URL.Query().Get(":db")
+	retentionPolicy := r.URL.Query().Get("retentionPolicy")
```

Comment:
Maybe just the RP, but you get the idea. Keeping the same as much as possible will prevent a lot of confusion.